src/testScripts/sdkV2-scripts.ts

```diff
@@ -125,6 +125,18 @@ async function addDeposits(network: Network) {
   // console.log('deposit stETH: ', txResponse.hash);
 }
 
+async function depositUsingESTGasLimit() {
```

Comment:
can keep this function here

just don't run it below by default. can comment it out and put it in the `main()` function